src/components/developer-preview.js

```diff
@@ -0,0 +1,288 @@
+            </tr>
+            <tr>
+              <td>/submissions</td>
+              <td>POST</td>
+              <td>Returns '401 Unauthorized'.</td>
+            </tr>
+            <tr>
+              <td>/measurement-sets</td>
+              <td>GET</td>
+              <td>Returns all measurement sets that you have submitted.</td>
+            </tr>
+            <tr>
+              <td>/measurement-sets/:id</td>
+              <td>GET</td>
+              <td>Returns measurement set if id belongs to you, otherwise '404 Not Found'.</td>
+            </tr>
+            <tr>
+              <td>/measurement-sets</td>
+              <td>POST</td>
+              <td>This endpoint allows to create a measurement set. The API will find/create the matching submission.</td>
+            </tr>
+            <tr>
+              <td>/measurement-sets/:id</td>
+              <td>PUT</td>
+              <td>This endpoint allows you to update measurement sets you have submitted (full replacement). Returns '404 Not Found' if id doesn't belong to you. If id is invalid or if id was not found.</td>
+            </tr>
+            <tr>
+              <td>/measurement-sets/:id</td>
+              <td>PATCH</td>
+              <td>This endpoint allows you to update measurement sets you have submitted (only replaces fields submitted). Returns '404 Not Found' if id doesn't belong to you or if id was not found.</td>
+            </tr>
+            <tr>
+              <td>/measurement-sets/:id</td>
+              <td>DELETE</td>
+              <td>This endpoint allows you to delete measurement sets you have submitted. Returns '404 Not Found' if id doesn't belong to you or if id was not found.</td>
+            </tr>
+            <tr>
+              <td>/measurements</td>
+              <td></td>
+              <td>Same as /measurement-sets</td>
+            </tr>
+          </tbody>
+        </table>
+
+        <h1 className='ds-h1' id='endpoints'>Endpoints</h1>
+        <p className='ds-text'>Visit the <a href='https://qpp-submissions-sandbox.navapbc.com/#!'>Interactive Documentation</a> to learn more about the endpoints available in the Submissions API.</p>
+        <p className='ds-text'>If you are a Registry or QCDR using the Developer Preview, you have an API key that is associated with an 'organization'. This affects what endpoints you are authorized to use, and what behavior each endpoint has.</p>
+        <p className='ds-text'>At a high level, your API key allows you to create and edit data using the /measurement-sets and /measurements endpoints. You cannot create data using the /submissions endpoint. For more information about what you're authorized to do with the Submissions API, click <a href='/qpp-submissions-docs/developer-preview#authorization'>here</a>.</p>
```

Comment:
should we make /measurement-sets , '/measurements and '/submissions' a different text to make it clear that those are API endpoints?